force-app/main/default/classes/BDI_Donations_TEST2.cls

```diff
@@ -127,17 +127,17 @@ public with sharing class BDI_Donations_TEST2 {
         }
 
         //Confirm that the DIs are Imported and that the correct status is in place for each Oppt/Payment
-        System.assertEquals(System.label.bdiImported,di1.Status__c);
+        System.assertEquals('Imported', di1.Status__c);
```

Comment:
Huh, is the DI logic hard coding the status to English?

Reply:
This is comparing a field value that doesn't get translated anyway. So this assert was failing since the label was being translated but the `Status__c` field value was not even in a non-English org.